inc/tools.php

```diff
@@ -227,6 +227,30 @@ function <?php echo esc_html( $callback ); ?>() {
  */
 function cptui_get_single_taxonomy_registery( $taxonomy = [] ) {
 
+	$might_need_to_resave = false;
+
+	$must_have_keys = ['object_types', 'rewrite', 'rewrite_slug', 'name',
```

Comment:
Actually perfect reason to make the keys function a reusable function. We can reduce code amount here.